### PR TITLE
updated formState api

### PR DESCRIPTION
**Pull request by liketurbo**
added isValid property to formState, it's nice to have one instead of calculating 🤗

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -475,6 +475,7 @@ export default function useForm<Data extends DataType>(
       submitCount: submitCountRef.current,
       touched: touchedFieldsRef.current,
       isSubmitting: isSubmittingRef.current,
+      isValid: Object.keys(errorsRef.current).length === 0
```
Comment (bluebill1049) [inline src/index.ts]: nice, can you use `isEmptyObject`?

**Comment by bluebill1049** [comment]
could you please add `invalid` type to types.ts?

File: src/index.ts (hunk 1 above)
Comment (liketurbo) [reply src/index.ts]: 😄, you doesn't have jquery or lodash in the project

**Comment by bluebill1049** [comment]
I will update the type :) 👍 thank you contributor

**Comment by liketurbo** [comment]
Not been sure what you mean by updating type in `types.ts`, if typescript can get type by ducktyping 🤔, wouldn't it be like overkill?
type

**Comment by bluebill1049** [comment]
yea you are right. that was my bad :(

**Comment by bluebill1049** [comment]
Screen Shot 2019-06-16 at 10 17 45 am
thank you ❤️

**Pull request merged**